src/uct/api/uct.h

```diff
@@ -526,7 +528,7 @@ ucs_status_t uct_iface_config_modify(uct_iface_config_t *config,
  * @param [in]  dev_name      Hardware device name,
  * @param [in]  rx_headroom   How much bytes to reserve before the receive segment.
  * @param [in]  config        Interface configuration options. Should be obtained
- *                            from uct_iface_read_config() function, or point to
+ *                            from uct_iface_config_read() function, or point to
```

Comment:
what do you mean? config is filled in uct_iface_config_read. this is not pd configuration, it's iface configuration.